XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+* If `AuthorizeCredentials` is included in the transaction:
+	* The credential(s) are already authorized.
+	* The account doesn't have enough reserve for the object.
+	* The array is too long (i.e. has more than 8 credentials).
+	* The array is empty (i.e. has no credentials).
+
+### 4.3. State Changes
+
+If the transaction is successful:
+* The `DepositPreauth` object is created or deleted.
+
+## 5. Transaction: `CredentialCreate`
+
+This transaction creates a `Credential` object.
+
+There are two possible methods of doing so:
+* The issuer submits this transaction, and `CredentialAccept` is used for the subject to accept the transaction.
+* The credential data is passed off-chain from the issuer to the subject. The subject then submits the credential themselves, alongside an issuer signature, to the ledger.
+
+### 5.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`TransactionType`| ✔️|`string`|`UInt16`|The transaction type (`CredentialCreate`).|
+|`Account`| ✔️|`string`|`AccountID`|The subject or issuer of the credential.|
+|`Subject`| |`string`|`AccountID`|The subject of the credential.|
+|`Issuer`| |`string`|`AccountID`|The issuer of the credential.|
+|`CredentialType`|✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+|`Signature`| |`string`|`Blob`|A signature from the issuer attesting to the credential.|
+|`Expiration`| |`number`|`UInt32`|Optional credential expiration.|
+|`URI`| |`string`|`Blob`|Optional additional data about the credential (such as a link to the VC document).|
+
+#### 5.1.1. `Subject` and `Issuer`
+
+Only one of these two fields must be specified. If `Subject` is specified, `Account` is the issuer. If `Issuer` is specified, `Account` is the subject.
+
+#### 5.1.2. `Signature`
+
+If the subject is submitting the account, a signature must be included from the issuer attesting to the credential. This signature will sign a simple JSON that looks something like this:
+```typescript
+{
+    Issuer: "rISABEL......",
+    Subject: "rALICE.......",
+    CredentialType: "123ABC",
+}
+```
+
+### 5.2. Failure Conditions
+
+* Both or neither `Subject` and `Issuer` are specified (i.e. there must be only one of the two fields).
+* The account in `Subject`/`Issuer` doesn't exist.
+* `Issuer` is included but `Signature` is not.
+* The time in `Expiration` is in the past.
+* The `URI` field is too long (limit 256 bytes).
+* The account doesn't have enough reserve for the object.
+
+### 5.3. State Changes
+
+If the transaction is successful:
+* The `Credential` object is created.
+* If `Issuer` == `Account`, then the `lsfAccepted` flag is enabled.
+
+## 6. Transaction: `CredentialAccept`
+
+This transaction transfers a credential from the `Issuer` to the `Account`. It is not a valid credential until it has been transferred.
```

Comment:
I wouldn't describe it as "transferring" the credential—especially if you follow my suggestion about keeping the ledger entry in both accounts' owner directories.

```suggestion
This transaction accepts a credential issued to the `Account`. It is not a valid credential until it has been accepted.
```

Reply:
Also: Maybe note that if you want to "un-accept" a credential you are supposed to delete it?